index.js

```diff
@@ -65,6 +65,12 @@ function expandPath(pathObj, defaultObj) {
     basedir = path.resolve(parsed.root);
   }
 
+  if (parsed.ext) {
+    filePath = filePath.replace(parsed.ext, '');
```

Comment:
If  the extension is `.js` and the directory or file title of `filePath` contains `.js`, this replacement does not return correct value.

Reply:
Thanks. I didn't think about that.  I'll add a test too.